src/types/modifications/FuelPrice.jl

```diff
@@ -20,7 +20,7 @@ Base.@kwdef struct FuelPrice <: Modification
 end
 export FuelPrice
 
-mod_rank(m::FuelPrice) = 0.0
+mod_rank(::Type{<:FuelPrice}) = 0.0
```

Comment:
Good catch.  Wouldn't it default to 0.0 anyway?  Curious where you saw something happening out of order?